WordPress/src/main/java/org/wordpress/android/ui/people/PeopleManagementActivity.java

```diff
@@ -97,4 +105,26 @@ public void onJSONException(JSONException exception) {
             }
         });
     }
+
+    @Override
+    public void onPersonSelected(Person person) {
+        FragmentManager fragmentManager = getFragmentManager();
+        PersonDetailFragment personDetailFragment = (PersonDetailFragment) fragmentManager.findFragmentByTag(KEY_PERSON_DETAIL_FRAGMENT);
+
+        long personID = person.getPersonID();
+        int localTableBlogID = person.getLocalTableBlogId();
+        if (personDetailFragment == null) {
+            personDetailFragment = PersonDetailFragment.newInstance(personID, localTableBlogID);
+        } else {
+            personDetailFragment.setPersonDetails(personID, localTableBlogID);
+        }
+        if (!personDetailFragment.isAdded()) {
+            FragmentTransaction fragmentTransaction = getFragmentManager().beginTransaction();
+            fragmentTransaction.setCustomAnimations(R.animator.fragment_slide_in_from_right, R.animator.fragment_slide_out_to_left,
```

Comment:
I noticed the effort to simulate an "activity" like style of animating in/out. At the end, there are a couple of issues with it:
* The animation is skipped if the device is rotate before tapping "Back"
* The "old" fragment slides out but that's not what the "old" activities are doing in the app (they just stay there on the background).
I know both of these issues are a bit beyond your control and will need ugly hacks to work around.

Instead, may I suggest resorting to `fragmentTransaction .setTransition(RANSIT_FRAGMENT_OPEN)` and let Android handle the animation? It will not look like an activity animating but that's prolly not a bad thing in this case. Let me know what you think! Cheers!